src/Unit-3/lesson3/README.md

```diff
@@ -0,0 +1,167 @@
+# Unit 3, Lesson 3 - Performing batch operations using commands!
+
+As you learned in the [previous lesson](../lesson2/README.md), RavenDB
+provides a low-level API that you can use to execute not trivial tasks.
+
+In this lesson, you will learn how to use commands to update or delete 
+a large amount of documents answering a certain criteria.
+
+## The need to batch operations 
+
+Sometimes you will need to update or delete a large amout of documents 
+answering a certain criteria. 
+
+Using SQL it would be easy. You just need to use the `UPDATE` and `DELETE` 
```

Comment:
This sentence, placed at the at begging of the lesson, suggests that it won't be so easy in RavenDB. That could discourage the reader a little bit.